doc/howtos/software-update-server.rst

```diff
@@ -0,0 +1,61 @@
+.. _software-update-server:
+
+Software Update: Building an Ostro |trade| OS  Repository
+#########################################################
+
+This technical note provides instructions supporting the creation
+of a repository for feeding updates to devices based on Ostro OS.
+
+Prerequisites
+=============
+
+- additional (fast) disk space: the SW Update tool is provided as
+  additional layer, therefore it doesn't require any additional
+  element.
+  The process of generating the bundles and updates can be very
+  disk intensive and dramatic speed improvements can come from
+  using a fast disk storage system, including use of SSD and RAID
+  arrays.
+  An execution round will also produce various rootfs directoriesy
+  as intermediate artefacts.
+  The actual disk footprint depends on the bundles selected and/or
+  defined, but it will definitely be larger, compared to the
+  generation of a simple build, like one might be used to, with a
+  typical distribution based on OECore/YP
+- signing keys (alternatively testing keys from the :file:`swupd-server`
+  project can be used).
+
+Bundles definition
+==================
+
+The extensive documentation for using SW Update in YP context can
+be found in the `swupd layer`_ and will be only summarized here.
+These instructions are based on the assumption that the work will
+happen on top of an image flavor that has swupd enabled .
+Example:
+``bitbake ostro-image-swupd`` or download one containing ``-swupd``
```

Comment:
... or download a pre-compiled image with ``-swupd`` in its name from https:/download.ostroproject.org